lib/topology_lib_ip/library.py

```diff
@@ -24,6 +24,130 @@
 
 from ipaddress import ip_address, ip_network, ip_interface
 
+import re
```

Comment:
I would be happy to. But I want to double check. Doing it that way (from re import match show) would be inconsistent with how it is implemented in ops-topology-lib-vtysh/lib/topology_lib_vrysh/parser.py which I was using as my example  to follow.

Let me know which way you want to go.